docker/node.docker-compose.yml

```diff
@@ -142,34 +142,34 @@ services:
     # ports: ['${PORTPREFIX}01:80']
     command: bash -c "cd /src/tests/helpers/servers &&
       FLASK_APP=dummy_test_helper
-      flask run --host=0.0.0.0 --port=80"
+      flask run --host=0.0.0.0"
 
   message_api:
     <<: *base_api_service
     container_name: ${COMPOSE_PROJECT_NAME}_ig_message_api
     depends_on:
       - elasticmq
       - minio
-    ports: ['${PORTPREFIX}01:80']
+    ports: ['${MESSAGE_API_BIND_HOST_PORT}:5000']
     command: bash -c "cd /src/intergov &&
       FLASK_APP=apis.message.app
-      flask run --host=0.0.0.0 --port=80"
+      flask run --host=0.0.0.0"
 
   subscriptions_api:
     <<: *base_api_service
     container_name: ${COMPOSE_PROJECT_NAME}_ig_subscriptions_api
-    ports: ['${PORTPREFIX}02:80']
+    ports: ['${SUBSCRIPTIONS_API_BIND_HOST_PORT}:5000']
     command: bash -c "cd /src/intergov &&
       FLASK_APP=apis.subscriptions.app
-      flask run --host=0.0.0.0 --port=80"
+      flask run --host=0.0.0.0"
 
   document_api:
     <<: *base_api_service
     container_name: ${COMPOSE_PROJECT_NAME}_ig_document_api
-    ports: ['${PORTPREFIX}03:80']
+    ports: ['${DOCUMENT_API_BIND_HOST_PORT}:5000']
     command: bash -c "cd /src/intergov &&
       FLASK_APP=apis.document.app
-      flask run --host=0.0.0.0 --port=80"
+      flask run --host=0.0.0.0"
```

Comment:
why not 80 port?

Reply:
It's only mapped to a host port binding anyway and it should break any instance where we are calling apis from within our own code. I believe we should use use_cases instead of calling our own APIs.